### ISSUE TITLE
Could not find region 'OUTOFSPACE' from atlas 

### ISSUE THREAD

**Issue by rpdelaney** [penguin0616/Insight#5]
After a longish hiatus I thought I'd pull out an old save. Got this:

```lua
[00:01:24]: [workshop-2081254154 (Insight)]:	Hunter has been hooked
[00:01:24]: WARNING! Could not find region 'OUTOFSPACE' from atlas 'images/inventoryimages.xml'. Is the region specified in the atlas?
[00:01:24]: images/inventoryimages.xml
LUA ERROR stack traceback:
        scripts/widgets/image.lua(30,1) in function 'SetTexture'
        scripts/widgets/image.lua(11,1) in function '_ctor'
        scripts/class.lua(181,1) in function 'Image'
        ../mods/workshop-662872357/scripts/widgets/foodingredientui.lua(45,1) in function '_ctor'
        scripts/class.lua(181,1) in function 'FoodIngredientUI'
        ../mods/workshop-662872357/scripts/widgets/foodrecipepopup.lua(255,1) in function '_InsertComponent'
        ../mods/workshop-662872357/scripts/widgets/foodrecipepopup.lua(141,1) in function '_CreateLayout'
        ../mods/workshop-662872357/scripts/widgets/foodrecipepopup.lua(77,1) in function '_ctor'
        scripts/class.lua(181,1) in function 'FoodRecipePopup'
        ../mods/workshop-662872357/scripts/widgets/fooditem.lua(23,1) in function '_ctor'
        scripts/class.lua(181,1) in function 'FoodItem'
        ../mods/workshop-662872357/scripts/widgets/foodcrafting.lua(57,1) in function 'OnAfterLoad'
        ../mods/workshop-662872357/modmain.lua(80,1) in function 'OnAfterLoad'
        ../mods/workshop-662872357/modmain.lua(99,1)
        =(tail call) ?
        =[C] in function 'xpcall'
        scripts/mods.lua(42,1)
        scripts/mods.lua(382,1) in function 'SetPostEnv'
        scripts/mainfunctions.lua(836,1) in function 'Check_Mods'
        scripts/mainfunctions.lua(829,1) in function 'CheckControllers'
        scripts/mainfunctions.lua(760,1)
```

The game then crashes to desktop. I'm scanning logs for more info that might be relevant.

Edit: I forgot to mention I also don't get the crash without Insight enabled.

**Comment by rpdelaney** [comment]
Lots of spam like these messages (I've redacted a couple hundred lines), then crash at the end:
```lua
[00:04:00]: scripts/mainfunctions.lua(192,1) Spawned followeritemslot from ../mods/workshop-344229670/modmain.lua:211
[00:04:00]: scripts/mainfunctions.lua(203,1) Entity hanging around origin:	(0.00, 0.00, 0.00)
[00:04:00]: scripts/mainfunctions.lua(192,1) Spawned followeritemslot from ../mods/workshop-344229670/modmain.lua:193
[00:04:00]: scripts/mainfunctions.lua(203,1) Entity hanging around origin:	(0.00, 0.00, 0.00)
[00:04:00]: scripts/mainfunctions.lua(192,1) Spawned followeritemslot from ../mods/workshop-344229670/modmain.lua:175
[00:04:00]: scripts/mainfunctions.lua(203,1) Entity hanging around origin:	(0.00, 0.00, 0.00)
[00:04:00]: scripts/mainfunctions.lua(192,1) Spawned followeritemslot from ../mods/workshop-344229670/modmain.lua:229
[00:04:00]: scripts/mainfunctions.lua(203,1) Entity hanging around origin:	(0.00, 0.00, 0.00)
[00:04:00]: Out of memory. Bytes requested: 1012964

Game removed: AppID 219740 "", ProcID 2745370
```

**Comment by penguin0616** [comment]
The first log seems to come from "workshop-662872357", which is [Craft Pot](https://steamcommunity.com/sharedfiles/filedetails/?id=662872357) (referenced in the stack trace)

The second one seems to come from "workshop-344229670", which is [Followers All in One v1.3](https://steamcommunity.com/sharedfiles/filedetails/?id=344229670) (referenced by "../mods/workshop-344229670/modmain.lua:229")

Can you try disabling those mods and seeing if you still crash?

Edit: Ah, I just saw your edit. Now that I double check the first log, it seems you posted a warning instead of the actual crash.
There should be a different error in the bottom?

**Comment by rpdelaney** [comment]
I will get you a more complete / verbose log soon hopefully, maybe if I fire up Don't Starve again tonight. :-)

**Comment by rpdelaney** [comment]
Full log, from booting steam to enabling the mod to crash: https://gist.github.com/rpdelaney/0d58556cd248eb1b9ecc6fcd715a940d

**Comment by penguin0616** [comment]
You seem to have the magic issue I have been unable to figure out the cause of. 
With no apparent error, Insight being active causes an out of memory error.
The biggest issue I have with this problem is that I haven't been able to reproduce it...

**Comment by rpdelaney** [comment]
I can send you the save file also if it would help, but there are many other mods I have enabled that you would need a few of them to load it.

**Comment by penguin0616** [comment]
Yeah, if you could provide your save that would be great.
I'm quite willing to download the other mods you have enabled, since I already use a fair amount of the mods you appear to be using.

**Comment by rpdelaney** [comment]
[All my saves](https://www.swisstransfer.com/d/2d8ab84e-c0b4-4c0b-ab2f-35043cbf3c4e). Link valid for 30 days. I'm getting the crash with the oldest one.

**Comment by penguin0616** [comment]
Today I learned of the .zst file. Praise WSL and stackoverflow. 
Here I go.

**Comment by penguin0616** [comment]
Okay, I figured out the issue. Pretty simple bandage fix, but I have to make sure the rest of my changes from DST won't crash in DS.

**Comment by rpdelaney** [comment]
zstandard is amazing!

**Comment by penguin0616** [comment]
Looks like it, yeah. I'll have to look into it more.
Thanks a bunch for providing the saves and logs, this has been an irritating bug.
_Also, nice base._

Fixed in be906adaa2c0955a8776040f728f3d4c3051be82 (mod version 2.9.5_ds)

**Comment by rpdelaney** [comment]
Confirmed this is working now, thank you!
